CHANGELOG.md

```diff
@@ -24,7 +24,6 @@ All notable changes to Gogs are documented in this file.
 - Configuration section `[service]` is no longer used.
 - Configuration option `APP_NAME` is no longer used.
 - Configuration option `[security] REVERSE_PROXY_AUTHENTICATION_USER` is no longer used.
-- Configuration option `[database] PASSWD` is no longer used.
```

Comment:
I don't think this is a valid deletion?

Reply:
Why not?

It seems duplicated:

![imagen](https://user-images.githubusercontent.com/1483414/97156945-b4fda180-1777-11eb-8c63-da4e53685358.png)

I can't see the difference... 😅

Reply:
Oh, OK, didn't see it in diffs 😅